Automattic/simplenote-android
# Issue/1043 crash web browser

### Fix
Update the `simperium` dependency from `0.9.12` to `0.9.13`, which includes adding browser detection to avoid a crash as described in https://github.com/Simperium/simperium-android/pull/225 and to close #1043.  When no browser is detected, a dialog is shown with instructions as well as a button to copy the URL.  A message is shown when copying the URL succeeds or fails.  See the screenshots below for illustration.

![crash_web_browser](https://user-images.githubusercontent.com/3827611/83668801-c7d7c580-a58d-11ea-95ee-fe8e58b79f85.png)

### Test
In order to get the ***No Browser Detected*** dialog to be shown, all browser apps must be uninstalled or disabled on the device before performing the following steps.  In order to test the ***Reset*** button in the ***Error*** dialog, change your Simplenote password to more than four and less than eight characters.

0. Clear app data.
1. Tap ***Sign Up*** button.
2. Tap ***Terms and Conditions*** link.
3. Notice ***No Browser Detected*** dialog is shown.
4. Tap ***OK*** button in dialog.
5. Tap back arrow in top app bar.
6. Tap ***Log In*** button.
7. Tap ***Log In with Email*** button.
8. Tap ***Forgot password?*** link.
9. Notice ***No Browser Detected*** dialog is shown.
10. Tap ***OK*** button in dialog.
11. Enter email address in ***Email*** field.
12. Notice ***Log In*** button is disabled.
13. Enter password of more than four and less than eight characters in ***Password*** field.
14. Notice ***Log In*** button is enabled after four characters are input.
15. Tap ***Log In*** button.
16. Notice ***Error*** dialog with password requirements is shown.
17. Tap ***Reset*** button in dialog.
18. Notice ***No Browser Detected*** dialog is shown.
19. Tap ***Copy URL*** button in dialog.
20. Notice ***URL copied to clipboard*** message is shown.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
These changes do not require release notes.
